doc/changelog.md

```diff
@@ -366,6 +366,10 @@
   the `stablehlo.custom_call` conversion pass.
   [(#488)](https://github.com/PennyLaneAI/catalyst/pull/488)
 
+* Remove a duplicated variable definition for `grad_p` and add it to `func_p`, which was 
+  lacking from it 
+  [(#553)](https://github.com/PennyLaneAI/catalyst/pull/553)
+
```

Comment:
this change doesn't have any user impact and don't need to add chnagelog entries for such PRs.